dejagnu/baseboards/arc-sim-nsimdrv.exp

```diff
@@ -30,9 +30,16 @@ load_generic_config "sim"
 load_base_board_description "basic-sim"
 
 # Any multilib options are set in an environment variable.
-set multilib_opts "$env(ARC_MULTILIB_OPTIONS)"
+if { [ info exists env(ARC_MULTILIB_OPTIONS) ] } {
+    set multilib_opts "$env(ARC_MULTILIB_OPTIONS)"
+} else {
+    set multilib_opts ""
```

Comment:
process_multilib_options sets the multilib flags. Not sure if we do not call it will not break something else. It seems all boards are calling this function regardless if they have or not flags to be processed. I would suggest to call it as well always.